### ISSUE TITLE
Control y and x axis variables

### ISSUE THREAD

**Issue by qalid7** [kassambara/ggcorrplot#10]
I've a large matrix and perhaps I don't need to show the correlations between each and every variable. Instead, for comparison purposes between groups, I'd rather have a set of variables on the x-axis and the same on y-axis. Something like this figure: 
![image](https://user-images.githubusercontent.com/35524818/39638128-21dbfb2e-4fbd-11e8-89ef-acd4eeed7fc4.png)
https://www.researchgate.net/figure/Heatmap-showing-pearson-correlation-coefficients-between-security-indices-vs-features_fig2_233836152

It looks like an R figure but I have failed to find a nice way of producing something like this. Any ideas?

**Comment by ffineis** [comment]
@qalid7 I think this is the same feature request as https://github.com/kassambara/ggcorrplot/issues/5 - you just want rectangular correlation heatmaps, right? Package maintainer has agreed to work on it

**Comment by qalid7** [comment]
@ffineis thanks! I will surely be waiting for that ...

Do you have any idea how to do this otherwise? I may need it sooner actually. It's seems like a lot simpler problem than the ones already presented in these packages.

**Comment by qalid7** [comment]
@Chia-KaiYang oh! that simple! Thanks so much man

**Comment by Chia-KaiYang** [comment]
@qalid7 no worries, I had experience on similar issues before :)

**Comment by IndrajeetPatil** [comment]
@kassambara I think this is also resolved!

**Comment by alexandruioanvoda** [comment]
@Chia-KaiYang I'm not sure if that works with the rest of the features. For example, when using the P-val plotting feature (with non-rectangular corr matrices), it throws an error, while without, it works just fine.

**Comment by acjmmartin** [comment]
@alexandruioanvoda were you able to fix your `Error in FUN(X[[i]], ...) : object 'Var1' not found` error? I am getting the same error when adding a p.mat matrix (even though it's square)

**Comment by kassambara** [comment]
Non-square (m×n) correlation matrices are supported via `ggcorrplot(cor(x, y))` with the default `type = "full"` — see #5 for a worked example. Closing as a duplicate of #5.
